### DISCUSSION TITLE
mdp, a password generator for personal use

### DISCUSSION THREAD

**Discussion by mlgarchery** [uxbox/uxbox#1236] [Show and tell]
Hello penpot users !

I've decided to do a version 2 of a tool I use for monthes now. I intend to do a mobile app (until now, it was only for desktop).
Here is the [visual prototype](https://design.penpot.app/#/view/f9950fa0-194b-11ec-a2bf-b2702b3a67d1?page-id=f9950fa1-194b-11ec-a2bf-b2702b3a67d1&index=0&share-id=b95f0300-1a45-11ec-9c71-4a8af25a7bed)
Let me know what you think of it, using the comment section in penpot of course ;). Edit: it seems it's not possible for anonymous users to comment for now. Anyway, you can comment here.

Thx for your feedbacks

**Comment by diacritica** [comment]
Hi @mlgarchery !
I'm sorry this went a bit under radar. I'm unsure if we contacted you via other methods.
As you know, commenting on other people's designs and prototypes is tricky and so we will try to avoid it as much as we can since we are very biased towards our own "tastes". But for the looks of your prototype, it seems you had a great time using the interactive prototype options and the design looks like you're reusing a lot of components. I'd invite you to try out the upcoming release since there will be some supernice transition animations for interactive prototypes... perhaps a v3 with that?
